assignments/array-methods.js

```diff
@@ -77,7 +90,20 @@ console.log(ticketPriceTotal);
 // Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
 
 // Problem 1
-
+const largeDonations = runners.filter(word => word.shirt_donations > 90);
+console.log(largeDonations);
 // Problem 2
 
-// Problem 3
\ No newline at end of file
+let companyName = runners [0];
+
+let company = [];
+
+runners.forEach(function (obj) {
+    names.push(`${obj.company_name}`);
+});
+console.log(companyName);
```

Comment:
The good news is the forEach you wrote is correct, syntax-wise. The less-than-good news is that the way this is written, everything from line 98 to 103 isn't doing anything.

You are assigning the first object in the runners array to companyName, then you are logging that object. 

Then, on line 102, you are pushing all the company_name values into the names array, which I assume, is initialized earlier. I highly doubt you want to do that. 

Here is what I think you were going for:
```suggestion
// A list of all the companies
let company = [];
runners.forEach(function (obj) {
    company.push(`${obj.company_name}`);
});
console.log(company);
```

And then remove the companyName variable all together.